osu.Server.Spectator.Tests/Multiplayer/MultiplayerHostOnlyQueueTests.cs

```diff
@@ -86,8 +86,8 @@ public async Task RoomHasNewPlaylistItemAfterMatchStartWithOneItem()
 
                 // Players received callbacks.
                 Receiver.Verify(r => r.PlaylistItemAdded(It.Is<MultiplayerPlaylistItem>(i => i.ID == newItem.ID)), Times.Once);
-                Receiver.Verify(r => r.PlaylistItemChanged(It.Is<MultiplayerPlaylistItem>(i => i.ID == oldItem.ID && i.Expired)), Times.Once);
-                Receiver.Verify(r => r.SettingsChanged(room.Settings), Times.Exactly(2));
+                Receiver.Verify(r => r.PlaylistItemChanged(It.Is<MultiplayerPlaylistItem>(i => i.ID == oldItem.ID && i.Expired)));
```

Comment:
Was there a particular reason for dropping the invocation count part of these verify calls?

Reply:
When finishing a playlist item there's two updates that occur - one to mark the item as expired, and another to reorder _all_ playlist items.
This may not be required anymore because expired items don't get reordered anymore - their `PlaylistOrder` value doesn't change.

That being said, I don't think the changes to this test are that important? The way I see it is they're testing implementation details rather than semantics, and I originally was going to go even further but left that for a future PR. Do you disagree with that?

Reply:
I don't necessarily disagree with dropping the invocation count checks; on one side they may be useful to ensure there are no unnecessary / misplaced calls, but it does make the tests more brittle to implementation details changing. I don't think I would object to removing, but I didn't really have a problem with them being here either. Not sure, I mostly just asked out of curiosity more than anything.

Reply:
Have reverted for the time being just to reduce the scope of changes.